devices/android/device-base

```diff
@@ -0,0 +1,47 @@
+{% set PROJECT = PROJECT|default("testcases/android/") %}
+{% extends PROJECT + "master.jinja2" %}
+
+{% set partitions_urls = partitions_urls|default({}) %}
+{% set partitions_names = partitions_names|default({}) %}
+{% set timeout_download = timeout_download |default(10) %}
```

Comment:
following the other timeouts we have they are in the format '*_timeout'

Reply:
updated